src/api/BaseApi.ts

```diff
@@ -1,10 +1,11 @@
 import axios from "axios";
 
-const BASE_URL = 'http://localhost:3000';
+const BASE_URL: string = 'http://localhost:3000';
```

Comment:
TypeScript will infer the type in this case - if you set a const to a string it will know it's string 😄

Try to remove the type here 👀